stetho/src/main/java/com/facebook/stetho/inspector/network/NetworkEventReporterImpl.java

```diff
@@ -152,6 +152,22 @@ public void responseHeadersReceived(InspectorResponse response) {
           getResourceTypeHelper().determineResourceType(contentType) :
           Page.ResourceType.OTHER;
       receivedParams.response = responseJSON;
+
+      AsyncPrettyPrinterRegistry registry= peerManager.getAsyncPrettyPrinterRegistry();
+      AsyncPrettyPrinterFactory factory = null;
+      for (int i = 0; i < response.headerCount(); i++) {
```

Comment:
recommend a for-loop pattern like this to reduce per-iteration overhead:
`for (int i = 0, count = response.headerCount(); i < count; i++) {`
(yeah, this can actually matter on android!)